lucene/BUILD.md

```diff
@@ -86,7 +80,4 @@ Please join the Lucene-User mailing list by visiting this site:
 Please post suggestions, questions, corrections or additions to this
 document to the lucene-user mailing list.
 
-This file was originally written by Steven J. Owens <puff@darksleep.com>.
-This file was modified by Jon S. Stevens <jon@latchkey.com>.
-
 Copyright (c) 2001-2020 The Apache Software Foundation.  All rights reserved.
```

Comment:
Do we need copyright notice here? (I recently modified this line at https://github.com/apache/lucene-solr/pull/1449/files for the first time since 2005.)